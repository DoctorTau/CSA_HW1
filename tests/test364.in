323
818 5163 6096 4389 7514 -3985 -3446 -1339 -3203 -949 -66 9262 -323 405 1127 -2369 -4087 212 6003 -2932 -4938 2223 2620 -4419 8488 5530 -8123 1492 -1834 6157 9855 -4104 7461 -2723 8191 7877 -4241 1374 4977 7943 4214 -4699 -7825 -2762 -436 2876 -8382 -8270 8775 -6078 9424 -9279 -8793 -4490 -1249 4037 4964 -8601 -7691 4342 788 3804 -6702 3792 7613 2667 -5172 -3878 2869 -4544 -5130 2604 2873 -7329 -5727 1794 -7828 -1773 -3681 -994 -6757 -3455 291 2311 -706 2373 -7150 3181 -1188 1569 6344 9803 5156 8880 -8612 9761 -1603 -910 -4299 1780 -8895 2981 -672 -8570 3302 -9678 -903 -2291 2055 -4699 -7528 3459 5812 6865 8087 8420 253 -5109 62 -3482 5026 9151 -3887 8089 -7447 -2789 -8615 5653 6835 -7333 535 -415 -8083 -1241 90 8281 -4368 9789 7325 -5080 1115 -7751 -9558 -4960 -7831 3010 3786 -4385 -1568 4286 4964 -8814 9806 3818 -5647 9673 2193 5469 5741 4953 2335 5708 -949 6765 -4292 -2998 -1165 -4283 -2896 6151 -6066 -580 -3191 8375 -5683 6483 -3393 2947 -3846 9925 -1031 4454 -3231 8260 4267 1078 2371 -184 -5536 -5836 -3388 5111 -8715 -7535 -6373 7202 -2958 -7295 -3297 7504 8217 5339 8210 -4272 1775 -489 880 -7695 -2203 8371 -5822 3644 -7340 7437 -6090 -876 -7174 -9273 -571 2660 5678 -7783 3330 7297 -5216 -2522 -7040 -9711 7908 -3541 3675 -2895 -9893 2332 3423 6650 -3453 -4604 -8085 9523 1866 -7801 -4208 1001 -2291 -9458 3542 -3250 7292 8557 -1617 -7655 5753 -9011 -3811 4287 -5096 -2070 -6031 8038 9016 -8753 2691 2052 -2305 -3070 -2233 -3214 3251 -1112 -104 7564 -2909 2255 -1734 7093 -6547 -8025 2866 9784 -4366 -8754 7623 2721 -6591 7826 4069 7758 1236 7794 4030 6291 9323 -1991 -8674 -3256 6929 -7250 6221 -5653 -100 -230 -7488 -1708 -1260 -7035 2542 2379 -9770 5405 2349 9931 -8050 1088 -6148 -1479 -5766 1673 -2362 8228 9555 8702 -3857 