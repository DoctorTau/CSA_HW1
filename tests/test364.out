-3446 -66 -2932 -4938 -1834 -4104 -2762 -436 -8382 -8270 -6078 -4490 -6702 -5172 -3878 -4544 -5130 -7828 -994 -706 -7150 -1188 -8612 -910 -672 -8570 -9678 -7528 -3482 -4368 -5080 -9558 -4960 -1568 -8814 -4292 -2998 -2896 -6066 -580 -3846 -184 -5536 -5836 -3388 -2958 -4272 -5822 -7340 -6090 -876 -7174 -5216 -2522 -7040 -4604 -4208 -9458 -3250 -5096 -2070 -3070 -3214 -1112 -104 -1734 -4366 -8754 -8674 -3256 -7250 -100 -230 -7488 -1708 -1260 -9770 -8050 -6148 -5766 -2362 